src/status_im2/common/resources.cljs

```diff
@@ -4,6 +4,8 @@
 (def ui
   {:add-new-contact               (js/require "../resources/images/ui2/add-contact.png")
    :biometrics                    (js/require "../resources/images/ui2/biometrics.png")
+   :this-is-not-keycard           (js/require "../resources/images/ui2/this-is-not-keycard.png")
+   :wrong-keycard           (js/require "../resources/images/ui2/wrong-keycard.png")
```

Comment:
Could we adjust the map alignment, please? Our [guidelines](https://guide.clojure.style/#map-keys-alignment) and [coding standards](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md) can help. You can also use 'make lint' & 'make lint-fix' commands to spot and fix issues.

Reply:
@ajayesivan thanks for the guidance. Fixed it!